integration-tests-ex/cases/src/test/resources/cluster/AzureDeepStorage/docker.yaml

```diff
@@ -0,0 +1,58 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#-------------------------------------------------------------------------
+
+# Definition of the batch index test cluster.
+# See https://yaml.org/spec/1.2.2 for more about YAML
+include:
+  - /cluster/Common/zk-metastore.yaml
+
+druid:
+  coordinator:
+    instances:
+      - port: 8081
+  overlord:
+    instances:
+      - port: 8090
+  broker:
+    instances:
+      - port: 8082
+  router:
+    instances:
+      - port: 8888
+  historical:
+    instances:
+      - port: 8083
+  indexer:
+    instances:
+      - port: 8091
+
+# Properties to be set in the Properties object used in
+# Guice configuration in lieu of the server-side runtime.properties
+# file.
+#
+# druid.global.http.numMaxThreads avoids creating 40+ Netty threads.
+# We only ever use 1.
+# druid.test.config.dockerIp is used by some older test code. Remove
+# it when that code is updated.
+properties:
+  druid.global.http.numMaxThreads: 3
```

Comment:
Most of these properties are not needed by this test. And, some of these are a bit of a confusion: these properties are for the test _client_, not the _server_. Only the thread settings are needed, and only to keep the client from creating a large number of threads which clutter the IDE experience unnecessarily.